packages/rancher-sachet/charts/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+annotations:
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/release-name: rancher-sachet
+  catalog.cattle.io/os: linux
+apiVersion: v2
+name: sachet
```

Comment:
This should be rancher-sachet, you'll need to update dependencies too.

Reply:
fixed.